packages/dcos-launch/extra/launch/onprem.py

```diff
@@ -0,0 +1,138 @@
+import copy
+import logging
+import subprocess
+
+import retrying
+import yaml
+
+import launch.aws
+import launch.util
+
+import test_util.aws
+import test_util.onprem
+import test_util.runner
+from test_util.helpers import Url
+
+log = logging.getLogger(__name__)
+
+STATE_FILE = 'LAST_COMPLETED_STAGE'
+
+
+@retrying.retry(wait_fixed=1000, stop_max_delay=120000)
+def check_ssh(ssher, host, port):
+    ssher.get_home_dir(host, port)
+
+
+class OnpremLauncher(launch.util.AbstractLauncher):
+    def __init__(self, config):
+        # can only be set during the wait command
+        self.bootstrap_host = None
+        self.config = config
+
+    def create(self, config):
```

Comment:
no, we don't need it so this was merged: https://github.com/dcos/dcos/pull/1485/commits/2c75b6868691cde37bc01c4d7f233789633640e9
:)